packages/perspective-jupyterlab/test/js/unit/view.spec.js

```diff
@@ -365,8 +372,14 @@ describe("PerspectiveView", function() {
                 const widget_mock = PerspectiveJupyterWidget.mock.instances[0];
                 const load_args = widget_mock.load.mock.calls[0];
 
-                expect(load_args[0]).toEqual(data);
-                expect(load_args[1]).toEqual(options);
+                const result = await load_args[0].view().to_columns();
+                result.b = result.b.map(x => new Date(x));
+
+                expect(result).toEqual(data);
+
+                // TODO there is no way to verify this in perspective API
+                // currently ...
+                // expect(load_args[1]).toEqual(options);
```

Comment:
We can use get_index() or get_limit() here to make sure it's been applied properly.